src/scss/_variables.scss

```diff
@@ -7,6 +7,10 @@
 /// @type Bool
 $o-card-is-silent: true !default;
 
+/// Silent mode
+/// @type Bool
+$o-fonts-is-silent: true !default;
```

Comment:
After @AlbertoElias's comments something like `$o-card-output-fonts`, would be better here as `$o-fonts-is-silent` will be used in `o-fonts`.